frontend/packages/dev-console/src/components/gitops/list/GitOpsListItem.tsx

```diff
@@ -22,7 +24,9 @@ const GitOpsListItem: React.FC<GitOpsListItemProps> = ({ appGroup }) => {
             <ResourceLink kind="application" name={appGroup.name} linkTo={false} />
           </GridItem>
           <GridItem lg={6} md={6} sm={6}>
-            {`${_.size(appGroup.environments)} Environments`}
+            {t('devconsole~{{enviromentSize, number}} Environments', {
+              enviromentSize: _.size(appGroup.environments),
+            })}
```

Comment:
Would be good to have it here as well:
```suggestion
            {t('devconsole~{{count, number}} Environment', {
              count: _.size(appGroup.environments),
            })}
```